packages/react/src/components/InAppMessaging/hooks/useMessageImage/__tests__/useMessageImage.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { ConsoleLogger as Logger } from '@aws-amplify/core';
+
+import useMessageImage from '../useMessageImage';
+
+type ImageConstructor = new (
+  width?: number | undefined,
+  height?: number | undefined
+) => HTMLImageElement;
+
+// use empty mockImplementation to turn off console output
+const errorSpy = jest.spyOn(Logger.prototype, 'error').mockImplementation();
+
+const src = 'https://test.jpeg';
+const image = { src };
+
+describe('useMessageImage', () => {
+  const original = global.Image;
+  afterAll(() => {
+    // restore Image
+    global.Image = original;
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('handles loading an image in the happy path as expected', async () => {
+    // set Image to mock succesful load
+    global.Image = class {
+      onload = jest.fn();
+      constructor() {
+        setTimeout(() => {
+          this.onload();
+        }, 50);
```

Comment:
Would these tests still pass if this were reduced to 0 (just a next tick)? We should always be weary of having timeout values during tests that might be fragile based on execution time.